packages/rich-text/src/formatters/convert.ts

```diff
@@ -62,15 +72,15 @@ function convertStyles(contentBlock, blockSelection, contentState) {
   return updatedContentState;
 }
 
-const styleMap = {
-  BOLD: weightToStyle('700'),
+const styleMap: Record<string, string> = {
+  BOLD: weightToStyle(700),
   ITALIC: ITALIC,
   UNDERLINE: UNDERLINE,
 };
 
-function getNewStyles(oldStyles) {
+function getNewStyles(oldStyles: string[]): string[] {
   return oldStyles
-    .map((oldStyle) => (oldStyle in styleMap ? styleMap[oldStyle] : null))
+    .map<string>((oldStyle) => styleMap[oldStyle] ?? null)
     .filter(Boolean);
 }
```

Comment:
> I don't know why that last cast is required, because if it's in the object, it must match one of the keys, so not too sure about that.

Yep, for some reason it's required when mapping, just saw this before elsewhere, too.

Sure, I'll go with your version, perhaps a bit cleaner indeed.